include/astshim/PolyMap.h

```diff
@@ -195,30 +195,30 @@ class PolyMap : public Mapping {
     @param[in] forward  If true the forward transformation is replaced.
                     Otherwise the inverse transformation is replaced.
     @param[in] acc  The target accuracy, expressed as a geodesic distance within
-                    the PolyMap's input space (if `forward` is true)
-                    or output space (if `forward` is false).
+                    the PolyMap's input space (if `forward` is false)
+                    or output space (if `forward` is true).
```

Comment:
Interesting. I don't find it so. I'm documenting it in the order "most common case first".